pkg/sql/catalog/rewrite/rewrite.go

```diff
@@ -86,31 +86,46 @@ func TableDescs(
 			return err
 		}
 
-		// Remap type IDs and sequence IDs in all serialized expressions within the TableDescriptor.
+		// Drop triggers if referenced tables, types, or routines not found.
+		dropTriggerMissingDeps(table, descriptorRewrites)
```

Comment:
I was following the pattern of check constraints + column expressions. It turns out we check that all referenced routines are present, and normally error out before we hit this `drop*MissingDeps` stage. There's a `SKIP_MISSING_UDFS` option that prevents the error, and we'll just drop the table elements that reference the routines instead. I'll add a test for this.

Reply:
It might be useful to add to the commit (or PR) text a note that the `SKIP_MISSING_UDFS` option applies to triggers too so docs can include that.

Reply:
>It might be useful to add to the commit (or PR) text a note that the SKIP_MISSING_UDFS option applies to triggers too so docs can include that.

Good idea, Done.